pkg/gofr/http/middleware/logger.go

```diff
@@ -140,6 +140,7 @@ func panicRecovery(re any, w http.ResponseWriter, logger logger) {
 	default:
 		e = "Unknown panic type"
 	}
+
```

Comment:
Maybe this line could be removed, but I tend do not commit such change, when there is nothing else to change in a file

It helps when looking at file history.

Of course, such change is fine if the MR purpose is of course about formatting

Reply:
<img width="1342" alt="Screenshot 2024-11-20 at 3 22 49 PM" src="https://github.com/user-attachments/assets/5a1b2a60-c21a-4ea4-bd32-7cf0e94bca65">

@ccoVeille removing that line will lead to a linter error😐